debian/changelog

```diff
@@ -1,3 +1,27 @@
+opx-nas-linux (5.25.0) unstable; urgency=medium
+
+  * Feature: VxLAN Support
+  * Bugfix: Fixed issue of dst_ip not set to system
+  * Bugfix: vrf_name is not present in the route/nbr CPS objects.
+  * Bugfix: Fix false netlink event for member addition during member deletion time
```

Comment:
Little confusing. If you can, please modify it.

Reply:
@rakeshdatta please make a suggestion

Reply:
My suggestion: `* Bugfix: Discard false netlink event for member addition during member deletion time`